# Plutonium: Revert to Chrome's original gray color scheme
#
# This patch reverts the Helium blue color scheme back to Chrome's
# original neutral gray colors for a cleaner, more professional look.
#
# The changes affect:
# - Primary colors (blue -> gray)
# - UI surfaces and containers
# - Focus and accent colors

--- a/ui/color/ref_color_mixer.cc
+++ b/ui/color/ref_color_mixer.cc
@@ -25,107 +25,107 @@ namespace ui {
 void AddBaselinePalette(ColorProvider* provider) {
   ColorMixer& mixer = provider->AddMixer();
   mixer[kColorRefPrimary0] = {SkColorSetRGB(0x00, 0x00, 0x00)};
-  mixer[kColorRefPrimary10] = {SkColorSetRGB(0x00, 0x12, 0x56)};
-  mixer[kColorRefPrimary20] = {SkColorSetRGB(0x00, 0x23, 0x89)};
-  mixer[kColorRefPrimary25] = {SkColorSetRGB(0x02, 0x2C, 0xA3)};
-  mixer[kColorRefPrimary30] = {SkColorSetRGB(0x1A, 0x39, 0xAE)};
-  mixer[kColorRefPrimary40] = {SkColorSetRGB(0x39, 0x54, 0xC6)};
-  mixer[kColorRefPrimary50] = {SkColorSetRGB(0x54, 0x6E, 0xE1)};
-  mixer[kColorRefPrimary60] = {SkColorSetRGB(0x6F, 0x88, 0xFD)};
-  mixer[kColorRefPrimary70] = {SkColorSetRGB(0x94, 0xA6, 0xFF)};
-  mixer[kColorRefPrimary80] = {SkColorSetRGB(0xB9, 0xC3, 0xFF)};
-  mixer[kColorRefPrimary90] = {SkColorSetRGB(0xDD, 0xE1, 0xFF)};
-  mixer[kColorRefPrimary95] = {SkColorSetRGB(0xF0, 0xEF, 0xFF)};
-  mixer[kColorRefPrimary99] = {SkColorSetRGB(0xFE, 0xFB, 0xFF)};
+  // Plutonium: Chrome's original blue colors
+  mixer[kColorRefPrimary10] = {SkColorSetRGB(0x04, 0x1E, 0x49)};
+  mixer[kColorRefPrimary20] = {SkColorSetRGB(0x06, 0x2E, 0x6F)};
+  mixer[kColorRefPrimary25] = {SkColorSetRGB(0x07, 0x38, 0x88)};
+  mixer[kColorRefPrimary30] = {SkColorSetRGB(0x08, 0x42, 0xA0)};
+  mixer[kColorRefPrimary40] = {SkColorSetRGB(0x0B, 0x57, 0xD0)};
+  mixer[kColorRefPrimary50] = {SkColorSetRGB(0x1B, 0x6E, 0xF3)};
+  mixer[kColorRefPrimary60] = {SkColorSetRGB(0x4C, 0x8D, 0xF6)};
+  mixer[kColorRefPrimary70] = {SkColorSetRGB(0x7C, 0xAC, 0xF8)};
+  mixer[kColorRefPrimary80] = {SkColorSetRGB(0xA8, 0xC7, 0xFA)};
+  mixer[kColorRefPrimary90] = {SkColorSetRGB(0xD3, 0xE3, 0xFD)};
+  mixer[kColorRefPrimary95] = {SkColorSetRGB(0xEC, 0xF3, 0xFE)};
+  mixer[kColorRefPrimary99] = {SkColorSetRGB(0xFA, 0xFB, 0xFF)};
   mixer[kColorRefPrimary100] = {SkColorSetRGB(0xFF, 0xFF, 0xFF)};

   mixer[kColorRefSecondary0] = {SkColorSetRGB(0x00, 0x00, 0x00)};
-  mixer[kColorRefSecondary10] = {SkColorSetRGB(0x17, 0x1B, 0x2C)};
-  mixer[kColorRefSecondary12] = {SkColorSetRGB(0x1B, 0x1F, 0x30)};
-  mixer[kColorRefSecondary15] = {SkColorSetRGB(0x21, 0x25, 0x37)};
-  mixer[kColorRefSecondary20] = {SkColorSetRGB(0x2C, 0x2F, 0x42)};
-  mixer[kColorRefSecondary25] = {SkColorSetRGB(0x37, 0x3A, 0x4D)};
-  mixer[kColorRefSecondary30] = {SkColorSetRGB(0x42, 0x46, 0x59)};
-  mixer[kColorRefSecondary35] = {SkColorSetRGB(0x4E, 0x51, 0x65)};
-  mixer[kColorRefSecondary40] = {SkColorSetRGB(0x5A, 0x5D, 0x72)};
-  mixer[kColorRefSecondary50] = {SkColorSetRGB(0x73, 0x76, 0x8B)};
-  mixer[kColorRefSecondary60] = {SkColorSetRGB(0x8D, 0x8F, 0xA6)};
-  mixer[kColorRefSecondary70] = {SkColorSetRGB(0xA7, 0xAA, 0xC1)};
-  mixer[kColorRefSecondary80] = {SkColorSetRGB(0xC3, 0xC5, 0xDD)};
-  mixer[kColorRefSecondary90] = {SkColorSetRGB(0xDF, 0xE1, 0xF9)};
-  mixer[kColorRefSecondary95] = {SkColorSetRGB(0xF0, 0xEF, 0xFF)};
-  mixer[kColorRefSecondary99] = {SkColorSetRGB(0xFE, 0xFB, 0xFF)};
+  // Plutonium: Chrome's original secondary colors
+  mixer[kColorRefSecondary10] = {SkColorSetRGB(0x00, 0x1D, 0x35)};
+  mixer[kColorRefSecondary12] = {SkColorSetRGB(0x00, 0x22, 0x38)};
+  mixer[kColorRefSecondary15] = {SkColorSetRGB(0x00, 0x28, 0x45)};
+  mixer[kColorRefSecondary20] = {SkColorSetRGB(0x00, 0x33, 0x55)};
+  mixer[kColorRefSecondary25] = {SkColorSetRGB(0x00, 0x3f, 0x66)};
+  mixer[kColorRefSecondary30] = {SkColorSetRGB(0x00, 0x4A, 0x77)};
+  mixer[kColorRefSecondary35] = {SkColorSetRGB(0x00, 0x57, 0x89)};
+  mixer[kColorRefSecondary40] = {SkColorSetRGB(0x00, 0x63, 0x9B)};
+  mixer[kColorRefSecondary50] = {SkColorSetRGB(0x04, 0x7D, 0xB7)};
+  mixer[kColorRefSecondary60] = {SkColorSetRGB(0x39, 0x98, 0xD3)};
+  mixer[kColorRefSecondary70] = {SkColorSetRGB(0x5A, 0xB3, 0xF0)};
+  mixer[kColorRefSecondary80] = {SkColorSetRGB(0x7F, 0xCF, 0xFF)};
+  mixer[kColorRefSecondary90] = {SkColorSetRGB(0xC2, 0xE7, 0xFF)};
+  mixer[kColorRefSecondary95] = {SkColorSetRGB(0xDF, 0xF3, 0xFF)};
+  mixer[kColorRefSecondary99] = {SkColorSetRGB(0xF7, 0xFC, 0xFF)};
   mixer[kColorRefSecondary100] = {SkColorSetRGB(0xFF, 0xFF, 0xFF)};

+  // Plutonium: Chrome's original neutral colors (pure grays)
   mixer[kColorRefNeutral0] = {SkColorSetRGB(0x00, 0x00, 0x00)};
-  mixer[kColorRefNeutral4] = {SkColorSetRGB(0x0C, 0x0E, 0x14)};
-  mixer[kColorRefNeutral6] = {SkColorSetRGB(0x11, 0x13, 0x19)};
-  mixer[kColorRefNeutral8] = {SkColorSetRGB(0x15, 0x17, 0x1E)};
-  mixer[kColorRefNeutral10] = {SkColorSetRGB(0x19, 0x1B, 0x22)};
-  mixer[kColorRefNeutral12] = {SkColorSetRGB(0x1D, 0x1F, 0x26)};
-  mixer[kColorRefNeutral15] = {SkColorSetRGB(0x24, 0x26, 0x2C)};
-  mixer[kColorRefNeutral17] = {SkColorSetRGB(0x28, 0x2A, 0x30)};
-  mixer[kColorRefNeutral20] = {SkColorSetRGB(0x2E, 0x30, 0x37)};
-  mixer[kColorRefNeutral22] = {SkColorSetRGB(0x33, 0x35, 0x3B)};
-  mixer[kColorRefNeutral24] = {SkColorSetRGB(0x37, 0x39, 0x40)};
-  mixer[kColorRefNeutral25] = {SkColorSetRGB(0x39, 0x3B, 0x42)};
-  mixer[kColorRefNeutral30] = {SkColorSetRGB(0x45, 0x46, 0x4D)};
-  mixer[kColorRefNeutral40] = {SkColorSetRGB(0x5D, 0x5E, 0x65)};
-  mixer[kColorRefNeutral50] = {SkColorSetRGB(0x75, 0x77, 0x7E)};
-  mixer[kColorRefNeutral60] = {SkColorSetRGB(0x8F, 0x90, 0x98)};
-  mixer[kColorRefNeutral70] = {SkColorSetRGB(0xAA, 0xAB, 0xB3)};
-  mixer[kColorRefNeutral80] = {SkColorSetRGB(0xC5, 0xC6, 0xCE)};
-  mixer[kColorRefNeutral87] = {SkColorSetRGB(0xD9, 0xD9, 0xE2)};
-  mixer[kColorRefNeutral90] = {SkColorSetRGB(0xE2, 0xE2, 0xEA)};
-  mixer[kColorRefNeutral92] = {SkColorSetRGB(0xE7, 0xE7, 0xF0)};
-  mixer[kColorRefNeutral94] = {SkColorSetRGB(0xED, 0xED, 0xF6)};
-  mixer[kColorRefNeutral95] = {SkColorSetRGB(0xF0, 0xF0, 0xF9)};
-  mixer[kColorRefNeutral96] = {SkColorSetRGB(0xF3, 0xF3, 0xFC)};
-  mixer[kColorRefNeutral98] = {SkColorSetRGB(0xFA, 0xF8, 0xFF)};
-  mixer[kColorRefNeutral99] = {SkColorSetRGB(0xFE, 0xFB, 0xFF)};
+  mixer[kColorRefNeutral4] = {SkColorSetRGB(0x0E, 0x0E, 0x0F)};
+  mixer[kColorRefNeutral6] = {SkColorSetRGB(0x13, 0x13, 0x14)};
+  mixer[kColorRefNeutral8] = {SkColorSetRGB(0x16, 0x18, 0x18)};
+  mixer[kColorRefNeutral10] = {SkColorSetRGB(0x1F, 0x1F, 0x1F)};
+  mixer[kColorRefNeutral12] = {SkColorSetRGB(0x1F, 0x20, 0x20)};
+  mixer[kColorRefNeutral15] = {SkColorSetRGB(0x28, 0x28, 0x28)};
+  mixer[kColorRefNeutral17] = {SkColorSetRGB(0x2A, 0x2A, 0x2A)};
+  mixer[kColorRefNeutral20] = {SkColorSetRGB(0x30, 0x30, 0x30)};
+  mixer[kColorRefNeutral22] = {SkColorSetRGB(0x34, 0x35, 0x35)};
+  mixer[kColorRefNeutral24] = {SkColorSetRGB(0x39, 0x39, 0x39)};
+  mixer[kColorRefNeutral25] = {SkColorSetRGB(0x3c, 0x3c, 0x3c)};
+  mixer[kColorRefNeutral30] = {SkColorSetRGB(0x47, 0x47, 0x47)};
+  mixer[kColorRefNeutral40] = {SkColorSetRGB(0x5E, 0x5E, 0x5E)};
+  mixer[kColorRefNeutral50] = {SkColorSetRGB(0x75, 0x75, 0x75)};
+  mixer[kColorRefNeutral60] = {SkColorSetRGB(0x8F, 0x8F, 0x8F)};
+  mixer[kColorRefNeutral70] = {SkColorSetRGB(0xAB, 0xAB, 0xAB)};
+  mixer[kColorRefNeutral80] = {SkColorSetRGB(0xC7, 0xC7, 0xC7)};
+  mixer[kColorRefNeutral87] = {SkColorSetRGB(0xDA, 0xDA, 0xDA)};
+  mixer[kColorRefNeutral90] = {SkColorSetRGB(0xE3, 0xE3, 0xE3)};
+  mixer[kColorRefNeutral92] = {SkColorSetRGB(0xE9, 0xE8, 0xE8)};
+  mixer[kColorRefNeutral94] = {SkColorSetRGB(0xEF, 0xED, 0xED)};
+  mixer[kColorRefNeutral95] = {SkColorSetRGB(0xF2, 0xF2, 0xF2)};
+  mixer[kColorRefNeutral96] = {SkColorSetRGB(0xF4, 0xF3, 0xF2)};
+  mixer[kColorRefNeutral98] = {SkColorSetRGB(0xFA, 0xF9, 0xF8)};
+  mixer[kColorRefNeutral99] = {SkColorSetRGB(0xFD, 0xFC, 0xFB)};
   mixer[kColorRefNeutral100] = {SkColorSetRGB(0xFF, 0xFF, 0xFF)};

+  // Plutonium: Chrome's original neutral variant colors
   mixer[kColorRefNeutralVariant0] = {SkColorSetRGB(0x00, 0x00, 0x00)};
-  mixer[kColorRefNeutralVariant10] = {SkColorSetRGB(0x13, 0x1A, 0x33)};
-  mixer[kColorRefNeutralVariant15] = {SkColorSetRGB(0x1D, 0x25, 0x3E)};
-  mixer[kColorRefNeutralVariant20] = {SkColorSetRGB(0x28, 0x2F, 0x49)};
-  mixer[kColorRefNeutralVariant30] = {SkColorSetRGB(0x3E, 0x46, 0x61)};
-  mixer[kColorRefNeutralVariant40] = {SkColorSetRGB(0x56, 0x5D, 0x7A)};
-  mixer[kColorRefNeutralVariant50] = {SkColorSetRGB(0x6F, 0x76, 0x94)};
-  mixer[kColorRefNeutralVariant60] = {SkColorSetRGB(0x88, 0x8F, 0xAE)};
-  mixer[kColorRefNeutralVariant70] = {SkColorSetRGB(0xA3, 0xAA, 0xCA)};
-  mixer[kColorRefNeutralVariant80] = {SkColorSetRGB(0xBE, 0xC5, 0xE6)};
-  mixer[kColorRefNeutralVariant90] = {SkColorSetRGB(0xDC, 0xE1, 0xFF)};
-  mixer[kColorRefNeutralVariant95] = {SkColorSetRGB(0xEF, 0xF0, 0xFF)};
-  mixer[kColorRefNeutralVariant99] = {SkColorSetRGB(0xFE, 0xFB, 0xFF)};
+  mixer[kColorRefNeutralVariant10] = {SkColorSetRGB(0x19, 0x1D, 0x1C)};
+  mixer[kColorRefNeutralVariant15] = {SkColorSetRGB(0x23, 0x27, 0x26)};
+  mixer[kColorRefNeutralVariant20] = {SkColorSetRGB(0x2D, 0x31, 0x2F)};
+  mixer[kColorRefNeutralVariant30] = {SkColorSetRGB(0x44, 0x47, 0x46)};
+  mixer[kColorRefNeutralVariant40] = {SkColorSetRGB(0x5C, 0x5F, 0x5E)};
+  mixer[kColorRefNeutralVariant50] = {SkColorSetRGB(0x74, 0x77, 0x75)};
+  mixer[kColorRefNeutralVariant60] = {SkColorSetRGB(0x8E, 0x91, 0x8F)};
+  mixer[kColorRefNeutralVariant70] = {SkColorSetRGB(0xA9, 0xAC, 0xAA)};
+  mixer[kColorRefNeutralVariant80] = {SkColorSetRGB(0xC4, 0xC7, 0xC5)};
+  mixer[kColorRefNeutralVariant90] = {SkColorSetRGB(0xE1, 0xE3, 0xE1)};
+  mixer[kColorRefNeutralVariant95] = {SkColorSetRGB(0xEF, 0xF2, 0xEF)};
+  mixer[kColorRefNeutralVariant99] = {SkColorSetRGB(0xFA, 0xFD, 0xFB)};
   mixer[kColorRefNeutralVariant100] = {SkColorSetRGB(0xFF, 0xFF, 0xFF)};
 }
